commit e5f2b577ded109291c9632dacb6eaa621d8a59fe
Author: Sylvain Gault <sylvain.gault@gmail.com>
Date:   Tue Sep 29 02:38:25 2015 +0200

    bios: Fix alignment change with gcc 5
    
    The section aligment specified in the ld scripts have to be greater or
    equal to those in the .o files generated by gcc.
    
    Signed-off-by: Sylvain Gault <sylvain.gault@gmail.com>
    Tested-by: poma <pomidorabelisima@gmail.com>
    Signed-off-by: Paulo Alcantara <pcacjr@zytor.com>

--- a/core/i386/syslinux.ld
+++ b/core/i386/syslinux.ld
@@ -266,7 +266,7 @@
 		__text_end = .;
 	}
 
-	. = ALIGN(16);
+	. = ALIGN(32);
 
 	__rodata_vma = .;
 	__rodata_lma = __rodata_vma + __text_lma - __text_vma;
@@ -361,7 +361,7 @@
 		__dynamic_end = .;
 	}
 
-	. = ALIGN(16);
+	. = ALIGN(32);
 
 	__data_vma = .;
 	__data_lma = __data_vma + __text_lma - __text_vma;
@@ -377,7 +377,7 @@
 	__pm_code_dwords = (__pm_code_len + 3) >> 2;
 
 	. = ALIGN(128);
-	
+
 	__bss_vma = .;
 	__bss_lma = .;		/* Dummy */
 	.bss (NOLOAD) : AT (__bss_lma) {
--- a/core/x86_64/syslinux.ld
+++ b/core/x86_64/syslinux.ld
@@ -266,7 +266,7 @@
 		__text_end = .;
 	}
 
-	. = ALIGN(16);
+	. = ALIGN(32);
 
 	__rodata_vma = .;
 	__rodata_lma = __rodata_vma + __text_lma - __text_vma;
@@ -361,7 +361,7 @@
 		__dynamic_end = .;
 	}
 
-	. = ALIGN(16);
+	. = ALIGN(32);
 
 	__data_vma = .;
 	__data_lma = __data_vma + __text_lma - __text_vma;
@@ -377,7 +377,7 @@
 	__pm_code_dwords = (__pm_code_len + 3) >> 2;
 
 	. = ALIGN(128);
-	
+
 	__bss_vma = .;
 	__bss_lma = .;		/* Dummy */
 	.bss (NOLOAD) : AT (__bss_lma) {
